### PR TITLE
Add azure private offer page

### PR DESCRIPTION
**Pull request by jpmartinspt**
## Done
Add azure private offer page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check [copy](https://docs.google.com/document/d/1ZgV1ZiyHJe3UNQkj0I-OCpZ4MIAWLHheig2Q2rclcuk/edit) matches

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9698

### CODE REVIEW

**Comment by anthonydillon** [comment]
I think you should add the page to navigation.yaml with the hidden flag. Otherwise you lose your place on the site.

I have asked in the issue if this should just be an engage page.

File: templates/azure/private-offer.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      width="143",
+      hi_def=True,
+      loading="auto"
+      ) | safe
+      }}
+    </a>
+  </div>
+  <div class="row">
+    <div class="col-7 u-vertically-center">
+      <h1>Ubuntu on Azure Private Offer</h1>
```
Comment (anthonydillon) [inline templates/azure/private-offer.html:25]:
```suggestion
      <h1>Ubuntu on Azure Private&nbsp;Offer</h1>
```

File: templates/azure/private-offer.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      <p class="u-no-padding--top p-heading--four">
+        Get exclusive services and support for your large scale linux operation
```
Comment (anthonydillon) [inline templates/azure/private-offer.html:27]:
```suggestion
        Get exclusive services and support for your large scale linux&nbsp;operation
```

File: templates/azure/private-offer.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{% extends "azure/base_azure.html" %}
+
+{% block title %}Ubuntu on Azure Private Offer{% endblock %}
```
Comment (anthonydillon) [inline templates/azure/private-offer.html:3]: This comes out as "Ubuntu on Azure Private Offer | Ubuntu". I think we can use the repetition here by changing to "Azure Private Offer"

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few suggestions in the code. The capitalisation concerns me but I assume this content has been approved.

**Comment by anasereijo** [comment]
@jpmartinspt can you update the demo link so I can access the page? I can't find it. Thanks

**Comment by anasereijo** [comment]
> @jpmartinspt can you update the demo link so I can access the page? I can't find it. Thanks

sorted. @anthonydillon sent me a link

**Comment by anasereijo** [comment]
- [ ] remove ubuntu logo

Screenshot 2021-05-06 at 11 57 25

**Comment by pmahnke** [comment]
From @campbieil we need to add a link to azure and he doesn't want it to have the Azure menu, so don't add it to navigation.yaml

File: templates/azure/private-offer.html (hunk 1 above)
Comment (jpmartinspt) [reply templates/azure/private-offer.html]: Added

File: templates/azure/private-offer.html (hunk 2 above)
Comment (jpmartinspt) [reply templates/azure/private-offer.html]: Added

File: templates/azure/private-offer.html (hunk 3 above)
Comment (jpmartinspt) [reply templates/azure/private-offer.html]: Updaded

**Comment by jpmartinspt** [comment]
@anasereijo @anthonydillon changes made.

**Review by anthonydillon — APPROVED** [review]
LGTM :+1: thanks

**Comment by anasereijo** [comment]
lgtm thanks

**Comment by lindseylopez** [comment]
Can we add the pro listing at the bottom please? https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic?tab=Overview

File: templates/azure/private-offer.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        <li class="p-list__item is-ticked">Exclusive levels of support</li>
+        <li class="p-list__item is-ticked">Upgraded security and compliance</li>
+        <li class="p-list__item is-ticked">Volume based pricing</li>
+      </ul>
+    </div>
+    <div class="col-5">
+      {% with id="3888", returnURL="https://ubuntu.com/azure/thank-you", cta="Contact Us" %}
+      {% include "engage/shared/_en_engage_form.html" %}
+      {% endwith %}
+    </div>
```
Comment (pmahnke) [inline templates/azure/private-offer.html:43]:
```suggestion
    </div>
    <p>
      <a class="p-link--external" href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic?tab=Overview">
        Visit the Azure Marketplace to download Ubuntu Pro
      </a>
    </p>
```

Comment (pmahnke) [reply templates/azure/private-offer.html:43]: @campbieil's text change

File: navigation.yaml [hunk 5]
```diff
@@ -15,6 +15,9 @@ azure:
     - title: Thank you
       path: /azure/thank-you
       hidden: True
+    - title: Private Offer
```
Comment (pmahnke) [inline navigation.yaml:18]: please remove these from here.

Comment (pmahnke) [reply navigation.yaml:18]: @jpmartinspt can you update these last two and push it live?

**Pull request merged**